app/models/manageiq/providers/embedded_ansible/automation_manager/job/status.rb

```diff
@@ -1,3 +1,24 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job::Status < ::OrchestrationStack::Status
-  include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Job::Status
+  attr_accessor :task_status
+
+  # This is a bit confusing, but because the OrchestrationStack::Status doesn't
+  # have a concept of both state and status as MiqTask does, inverting is the
+  # "DRYest" way to match up with `normalized_status` found in
+  # OrchestrationStack::Status.
```

Comment:
Note:  As the comment suggests, this changed quite a bit from what @Fryguy did first time around.  Specifically, the `::OrchestrationStack::Status#normalized_status` (which is now tested) was not returning as expected since `status` was just the `MiqTask` object itself, and not a string as expected.

But switching to overriding the `initialize`, I was able to mirror better what existed previously, and still being true to what Jason did the first time around.

Reply:
Yeah, I just hacked the interface to ::OrchestrationStack::Status to take an MiqTask, because it just didn't have enough information.  I like your way better.